### ISSUE TITLE
Tensorflow 1.1 and Python 3.6 under Windows not compatible (+solution)

### ISSUE THREAD

**Issue by tobigithub** [tobigithub/tensorflow-deep-learning#17]
As of June 2017 TF under Windows only supports Python 3.5 which was released two years ago August 25, 2015. In order to use CUDA, Tensorflow under Windows existing python versions have to be downgraded, best with version managers such as pip or anaconda or virtual environments like bezel.

Error when installing TF 1.1.0 under Windows and Anaconda Python 3.6:

````
Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users\XXX>conda install -c conda-forge tensorflow=1.0.0
Fetching package metadata .............
Solving package specifications: ....


UnsatisfiableError: The following specifications were found to be in conflict:
  - python 3.6*
  - tensorflow 1.0.0*
Use "conda info <package>" to see the dependencies for each package.



C:\Users\XXX>conda create -n tensorflow
Fetching package metadata ...........
.Solving package specifications: .
Package plan for installation in environment C:\Anaconda3\envs\tensorflow:

The following empty environments will be CREATED:

C:\Anaconda3\envs\tensorflow

Proceed ([y]/n)? y

#
# To activate this environment, use:
# > activate tensorflow
#
# To deactivate this environment, use:
# > deactivate tensorflow
#
# * for power-users using bash, you must source
#


C:\Users\XXX> activate tensorflow

(tensorflow) C:\Users\XXX>pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/windows/cpu/tensorflow-1.1.0-cp35-cp35m-win_amd64.whl
tensorflow-1.1.0-cp35-cp35m-win_amd64.whl is not a supported wheel on this platform.

(tensorflow) C:\Users\XXX>pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/windows/gpu/tensorflow_gpu-1.1.0-cp35-cp35m-win_amd64.whl
tensorflow_gpu-1.1.0-cp35-cp35m-win_amd64.whl is not a supported wheel on this platform.

(tensorflow) C:\Users\XXX>
````

Solution by AlexisTM see here https://github.com/tensorflow/tensorflow/issues/6999
````
conda create --name tensorflow python=3.5
activate tensorflow
conda install jupyter
conda install scipy
pip install tensorflow
# or
# pip install tensorflow-gpu
````
Or by using pip:
http://stackoverflow.com/questions/40884668/installing-tensorflow-on-windows-python-3-6-x

**Comment by oldmud0** [comment]
Not really a solution, just a workaround since you're still not using Python 3.6 as you intended.

**Comment by AlexisTM** [comment]
Indeed this is a workaround, but that allows you to use TF without a lot of configurations, as all dependencies are automatically installed and you have a working environment. 

Afterwards, if you want to deploy on a linux machine with Python3.6, you still can do it.

**Comment by oldmud0** [comment]
Anyway, the workaround you provided has already been well documented. Thanks anyway.

**Comment by tobigithub** [comment]
@oldmud0: true, has nothing to do with python 3.6

**Comment by tobigithub** [comment]
Closed, because not a solution to use Python 3.6 and tensorflow.
